### PR TITLE
Emilhotkowski/fallback without circuit

### PR DESCRIPTION
**Pull request by emilhotkowski**
### Description

Removed circuit breaker and used much simpler solution

### Tested

Run e2e tests. Unit tests are in place

### CODE REVIEW

File: packages/phone-number-privacy/combiner/src/signing/get-threshold-signature.ts [hunk 1]
```diff
@@ -314,28 +311,12 @@ function requestSignature(
   controller: AbortController,
   logger: Logger
 ): Promise<FetchResponse> {
-  if (!signersCircuitBreaker.has(service.url)) {
-    const options: Options = {
-      timeout: config.odisServices.timeoutMilliSeconds / 2,
-      resetTimeout: 15,
+  return parametrizedSignatureRequest(service.url, request, controller, logger).catch((e) => {
```
Comment (codyborn) [inline packages/phone-number-privacy/combiner/src/signing/get-threshold-signature.ts]: Nit
```suggestion
  return parameterizedSignatureRequest(service.url, request, controller, logger).catch((e) => {
```

File: packages/phone-number-privacy/combiner/src/signing/get-threshold-signature.ts [hunk 2]
```diff
@@ -314,28 +311,12 @@ function requestSignature(
...
+    if (service.fallbackUrl) {
```
Comment (codyborn) [inline packages/phone-number-privacy/combiner/src/signing/get-threshold-signature.ts:316]: It'd be great to log that the primary failed and we hit the fallback.

**Review by codyborn — APPROVED** [review]
LGTM!

File: packages/phone-number-privacy/combiner/src/signing/get-threshold-signature.ts (hunk 1 above)
Comment (emilhotkowski) [reply packages/phone-number-privacy/combiner/src/signing/get-threshold-signature.ts]: Changed, thx!

File: packages/phone-number-privacy/combiner/src/signing/get-threshold-signature.ts (hunk 2 above)
Comment (emilhotkowski) [reply packages/phone-number-privacy/combiner/src/signing/get-threshold-signature.ts:316]: Added

**Pull request merged**